signer/src/context/signer_context.rs

```diff
@@ -90,6 +91,13 @@ where
         if let Some(height) = config.signer.sbtc_bitcoin_start_height {
             state.set_sbtc_bitcoin_start_height(height);
         }
+        state.update_current_limits(SbtcLimits::new(
+            Some(Amount::ZERO),
+            Some(Amount::ZERO),
+            Some(Amount::ZERO),
+            Some(Amount::ZERO),
+            Some(Amount::ZERO),
```

Comment:
Don't we want the minimums to be `MAX_MONEY`? Actually, I support it doesn't matter.

Reply:
Yeah that's better, changed.